test/test_scripts/test_ufbrw_test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Do we need this script? Can the program just be run from CMake like the other tests?

Reply:
Yup, looks like it can. Working on it...

Reply:
Done and working. Just need a little cleanup and a doc block for it.